charts/nexus-repository-manager/values.yaml

```diff
@@ -115,7 +115,7 @@ service:
   enabled: true
   labels: {}
   annotations: {}
-  serviceType: ClusterIP
+  type: ClusterIP
```

Comment:
for others' reference. this correction aligns to the template, so the value actually gets used: https://github.com/sonatype/helm3-charts/blob/main/charts/nexus-repository-manager/templates/service.yaml#L19